### ISSUE TITLE
Unknown packaging: zip

### ISSUE THREAD

**Issue by mturra-Pro** [CycloneDX/cyclonedx-maven-plugin#36]
I have a type zip dependency.

```
<dependency>
        <groupId>com.proj.risk</groupId>
        <artifactId>util_common</artifactId>                       
        <version>1.0</version>
        <type>zip</type>
</dependency>
```

This command fails 
`mvn org.cyclonedx:cyclonedx-maven-plugin:makeBom`

with this message:
> [ERROR] Some problems were encountered while processing the POMs:
> [ERROR] Unknown packaging: zip @ line 6, column 14

Eventually is there any option to exclude zip packages?

**Comment by stevespringett** [comment]
At the moment, no, excluding specific types cannot be done. The correct behaviour should be to include the zip component but as component type=file instead of type=library.

I'll see what can be done to do this in the short term.

**Comment by stevespringett** [comment]
A new parameter, excludeTypes, was introduced which should solve this issue. excludeTypes is an optional array which allows you to specify Maven artifact types to exclude from the bom.

**Comment by stevespringett** [comment]
Resolved in v1.5.0
